pkg/helpers/helpers.go

```diff
@@ -173,12 +191,10 @@ func UpdateKlusterletConditionFn(conds ...metav1.Condition) UpdateKlusterletStat
 	}
 }
 
-// TODO: just for upgrading, we need to remove this in 0.11.0
-func ReplaceKlusterletConditionFn(removeConditionType string, conds ...metav1.Condition) UpdateKlusterletStatusFunc {
+func RemoveKlusterletConditionFn(condTypes ...string) UpdateKlusterletStatusFunc {
```

Comment:
this should be removed in 0.11.0, right？

Reply:
yes